qa/tasks/vstart_runner.py

```diff
@@ -773,6 +773,28 @@ def list_connections():
 
         self.gather_mount_info()
 
+    def _set_fuse_daemon_pid(self):
+        # NOTE: When a command <args> is launched with sudo, two processes are
+        # launched, one with sudo in <args> and other without. Make sure we
+        # get the PID of latter one.
+        for i in range(15):
+            time.sleep(1)
+
+            try:
+                sock = self.find_admin_socket()
+            except (RuntimeError, CommandFailedError):
+                continue
+
+            try:
+                pid = re.match(".*\.(\d+)\.asok$", sock).group(1)
```

Comment:
You could also use the `status` command to learn the pid.

Reply:
Okay, that might be easier.

Reply:
I would need to parse the output instead of asok filename which would more or less be same. So I'm skipping this refactoring.